core/adapters/http.go

```diff
@@ -89,11 +93,18 @@ func (hpa *HTTPPost) GetRequest(body string) (*http.Request, error) {
 	if err != nil {
 		return nil, err
 	}
+	appendExtendedPath(request, hpa.ExtPath)
 	appendQueryParams(request, hpa.QueryParams)
 	setHeaders(request, hpa.Headers, "application/json")
 	return request, nil
 }
 
+func appendExtendedPath(request *http.Request, extPath ExtendedPath) {
+	for _, element := range extPath {
+		request.URL.Path = path.Join(request.URL.Path, element)
```

Comment:
You don't need to iterate in a for loop here because `path.Join` takes a variadic parameter: https://golang.org/pkg/path/#Join

```
request.URL.Path = path.Join(request.URL.Path, []string(extPath)...)
```

https://stackoverflow.com/a/17556146/639773